.vib/grafana/goss/grafana.yaml

```diff
@@ -0,0 +1,29 @@
+command:
+  check-grafana-plugins:
+    exit-status: 0
+    exec: grafana-cli --pluginsDir /opt/bitnami/grafana/default-plugins/ plugins ls
+    stdout:
+      - agenty-flowcharting-panel
+      - briangann-datatable-panel
+      - briangann-gauge-panel
+      - digiapulssi-organisations-panel
+      - grafana-clock-panel
+      - grafana-piechart-panel
+      - grafana-polystat-panel
+      - jdbranham-diagram-panel
+      - larona-epict-panel
+      - marcuscalidus-svg-panel
+      - michaeldmoore-annunciator-panel
+      - michaeldmoore-multistat-panel
+      - natel-discrete-panel
+      - neocat-cal-heatmap-panel
+      - pierosavi-imageit-panel
+      - scadavis-synoptic-pane
+      - snuids-trafficlights-panel
+      - vonage-status-panel
```

Comment:
For readability's sake, could you please move this list to the vars file and check it here using a loop? We do something similar for Apache's modules.